Modules/zipimport.c

```diff
@@ -379,21 +405,29 @@ find_loader(ZipImporter *self, PyObject *fullname, PyObject **namespace_portion)
     return FL_MODULE_FOUND;
 }
 
+/*[clinic input]
+zipimport.zipimporter.find_module
+
+    fullname: unicode
```

Comment:
The arguments of this method (and many other methods) were positional-only. Converting to Argument Clinic shouldn't change the behavior.